tests/unit/altimeter/aws/resource/iam/test_role.py

```diff
@@ -33,3 +37,91 @@ def test_disappearing_role_race_condition(self):
             )
             resources = IAMRoleResourceSpec.scan(scan_accessor=scan_accessor)
             self.assertEqual(resources, [])
+
+    @mock_iam
+    def test_get_embedded_policy(self):
+        account_id = "123456789012"
+        region_name = "us-east-1"
+        role_name = "foo"
+
+        role_policy = "foo_policy"
+        policy_document = """{
+            "Statement": [
+                {
+                    "Action": ["sts:assumeRole"],
+                    "Effect": "Allow", "Resource": ["*"]
+                }
+            ],
+            "Version": "2012-10-17"
+        }"""
+
+        role_policy2 = "foo_policy2"
+        policy_document2 = """{
+            "Statement": [
+                {
+                    "Action": ["sqs:queue"],
+                    "Effect": "Allow", "Resource": ["*"]
+                }
+            ],
+            "Version": "2012-10-17"
+        }"""
+
+        assume_role_policy_document = """{
+            "Version": "2012-10-17",
+            "Statement": [
+                {
+                "Effect": "Allow",
+                "Principal": {
+                    "Service": "lambda.amazonaws.com"
+                },
+                "Action": "sts:AssumeRole"
+                }
+            ]
+        }"""
+        session = boto3.Session()
+        client = session.client("iam")
+        client.create_role(RoleName=role_name, AssumeRolePolicyDocument=assume_role_policy_document)
+        iam = boto3.resource('iam')
+        policy = iam.RolePolicy(role_name,role_policy)
+        policy.put(PolicyDocument= policy_document)
+        policy = iam.RolePolicy(role_name,role_policy2)
+        policy.put(PolicyDocument= policy_document2)
+
+        scan_accessor = AWSAccessor(session=session, account_id=account_id, region_name=region_name)
+        resources = IAMRoleResourceSpec.scan(scan_accessor=scan_accessor)
+        embedded_resources_links = [
+            link for link in resources[0].link_collection.multi_links
+            if link.pred == "embedded_policy"
+        ]
+        self.assertEqual(len(embedded_resources_links), 2)
+        # First policy.
+        embedded_resources_link = embedded_resources_links[0]
+        self.assertTrue(compare_embedded_policy(embedded_resources_link, role_policy, policy_document))
+        # Second policy.
+        embedded_resources_link = embedded_resources_links[1]
+        self.assertTrue(compare_embedded_policy(embedded_resources_link, role_policy2, policy_document2))
+
+def compare_embedded_policy(
+    source_policy,
+    expected_policy_name,
+    expected_policy_document
+    ):
```

Comment:
What's the maximum line width for this project? If it fits, I'd probably put the whole function signature in one line.

Reply:
This is the same case as the above I'm not entirely sure but I'm guessing is 100, so I will leave as it is.